spock-specs/src/test/groovy/org/spockframework/smoke/extension/StepwiseExtension.groovy

```diff
@@ -19,7 +19,7 @@ import org.junit.platform.engine.discovery.DiscoverySelectors
 import org.spockframework.EmbeddedSpecification
 
 class StepwiseExtension extends EmbeddedSpecification {
-  def "basic usage"() {
+  def "[spec] basic usage"() {
```

Comment:
That prefix grouping doesn't really fit with the rest of the specs, if we want to group something then the usual pattern is two have two separate specs, e.g. `StepwiseExtensionSpecs`/`StepwiseExtensionFeatures`
See `org.spockframework.smoke.extension.IncludeExclude*`

Reply:
Done in 8d293de.